PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/survey/CustomerSurveyHelper.java

```diff
@@ -63,7 +63,7 @@
 	private static final int INIT_SURVEY_DELAY_BY_DAY = 10;
 	private static final int PUT_OFF_DELAY_BY_DAY = 30;
 	private static final int TAKE_SURVEY_DELAY_BY_DAY = 180;
-	private static final int DISPOSE_TIME = 10;
+	private static final int DISPOSE_TIME = 20;
```

Comment:
Please add a comment here that Eclipse pop up window can't reset dispost time, so use a longer value here which differs  IntelliJ

Reply:
done